### DISCUSSION TITLE
Integrate real time debugger issue(Integrate Segger RTT viewer with STM32F411RE in eclipse ide)

### DISCUSSION THREAD

**Discussion by Sami2098** [eclipse-cdt/cdt#869] [Q&A]
hello all,
I was trying to integrate Segger RTT viewer with STM32F411RE in eclipse ide
all time when I connect RTT viewer this error occure in RTT viewer

"ERROR: <All Terminals>: Terminal change character (IAC) followed by invalid character (non-hexadecimal)."

I also attach debugger configuration

![Debug_config](https://github.com/user-attachments/assets/67330000-0d08-49f5-940e-dd49ecd60273)

**Comment by jonahgraham** [comment]
Hi @Sami2098 the OpenOCD plug-ins are from our sister project, embed-cdt. Please start a discussion there to get the attention of the folks best able to help you: https://github.com/eclipse-embed-cdt/eclipse-plugins/discussions
